### PR TITLE
LRQA-53659

### PR DESCRIPTION
**Pull request by Adriverforlife**
I created a new path file because I could not find one for the page I needed. If one exists and I just missed it, I can change the path on the check at the end of this test. It looks like the next couple tests are going to be using this page.

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase [hunk 1]
```diff
@@ -761,6 +761,41 @@ definition {
...
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry2 Content",
+			entryTitle = "Blogs Entry2 Title");
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry1 Content",
+			entryTitle = "Blogs Entry Title");
+
+		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+		Click(locator1 = "WorkflowMetrics#DASHBOARD_TOTAL_PENDING_ITEMS_All_ITEMS");
```
Comment (barrosdennys) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase]: hi @Adriverforlife ,
Is DASHBOARD_TOTAL_PENDING_ITEMS_VALUE not applicable for this case?

**Comment by barrosdennys** [comment]
@Adriverforlife ,
Hi Michael, we have WorkflowAllItems.path 

I believe this file can be used for this test.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase (hunk 1 above)
Comment (Adriverforlife) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase]: You are right. In my manual testing I missed that you could click anywhere in that box.I swapped this around and got rid of the unneeded path.

**Comment by Adriverforlife** [comment]
@barrosdennys did the fix above and moved the path I needed to WorlfowAllItems. I still had to use the pathing I was using though, as the test call for making sure the results show up in a particular order.

**Comment by barrosdennys** [comment]
Sent to BChan:
https://github.com/barrosdennys/liferay-portal/pull/89

**Pull request closed without merging**